packages/scene-composer/src/components/panels/SceneNodeInspectorPanel.tsx

```diff
@@ -97,6 +102,26 @@ export const SceneNodeInspectorPanel: React.FC = () => {
     }
   };
 
+  const scrollToBottom = useCallback(() => {
+    // Use setTimeout to let page updates with the new component first
+    setTimeout(() => {
+      // TODO: better find the element to scroll
+      let scrollElement: HTMLElement | null = containerRef.current;
+
+      while (scrollElement) {
+        if (scrollElement.className.includes(TABBED_PANEL_CONTAINER_NAME)) {
+          break;
+        }
+        scrollElement = scrollElement.parentElement;
+      }
+
+      scrollElement?.parentElement?.scroll({
+        top: scrollElement.scrollHeight,
+        behavior: 'smooth',
+      });
+    }, 5);
```

Comment:
Non-blocker: This section could benefit from refactoring or comments to aid understanding. Please improve this forward.

Reply:
comments added